Readme.md

```diff
@@ -1,5 +1,7 @@
 # Lyef React Counter
 [![Build Status](https://travis-ci.org/lyef/lyef-react-counter.svg)](https://travis-ci.org/lyef/lyef-react-counter)
+![](https://badge-size.herokuapp.com/lyef/lyef-react-counter/master/dist/Counter.min.js.svg)
```

Comment:
Put some alt here, it's better to accessibility and just in case if doesn't render the badge will show what is.